src/libswami/SwamiLog.h

```diff
@@ -70,10 +70,10 @@ extern void _swami_pretty_log_handler (GLogLevelFlags flags,
 	 	 	     __FILE__, __PRETTY_FUNCTION__, __LINE__, \
 	 	 	     format, ## args);
 #else
-#define SWAMI_DEBUG(format, args...)
+#define SWAMI_DEBUG(format, args)
 #endif
 
-#define SWAMI_INFO(format, args...) \
+#define SWAMI_INFO(format, args) \
```

Comment:
Variadic macros are a C99 feature. You need VS2015 to use it. Your change breaks compilation I'm afraid. You may convert these macros to normal functions though, to make it work.